packages/worker/src/workflow/bundler.ts

```diff
@@ -183,14 +183,20 @@ exports.importInterceptors = function importInterceptors() {
     entry: string,
     distDir: string
   ): Promise<string> {
-    const captureProblematicModules: Configuration['externals'] = async (data, _callback): Promise<undefined> => {
+    const captureProblematicModules: Configuration['externals'] = async (
+      data,
+      _callback
+    ): Promise<string | undefined> => {
       // Ignore the "node:" prefix if any.
       const module: string = data.request?.startsWith('node:')
         ? data.request.slice('node:'.length)
         : data.request ?? '';
 
-      if (moduleMatches(module, disallowedModules) && !moduleMatches(module, this.ignoreModules)) {
-        this.foundProblematicModules.add(module);
+      if (moduleMatches(module, disallowedModules)) {
+        if (!moduleMatches(module, this.ignoreModules)) {
+          this.foundProblematicModules.add(module);
+        }
+        return 'var {}';
```

Comment:
Should we change to return an empty object? Is this better experience in your opinion?

Reply:
Not sure what you mean... Do you mean `var false` vs `var {}`?

Reply:
We used to return `undefined` here.

Reply:
Oh, I see... Actually, THIS is the fix.

By returning `var {}`, we are telling Webpack to replace imports of that module by an empty object. This is essentially the same as what we do with `alias`, but `externals` is called much earlier in the process, so that we can still handle invalid namespaces such as `node:`.